### PR TITLE
Added textInput stories

### PR DESCRIPTION
**Pull request by raquelarrojo**
- Added textInput stories.
- Added ellipsis to input and suggestions.

### CODE REVIEW

File: lib/src/text-input/TextInput.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,418 @@
...
+import DxcTextInput from "./TextInput";
+
+export default {
+  title: "Text input",
+  component: DxcTextInput,
+};
+
+const action = {
+  onClick: () => {
+    console.log("Copy that!");
```
Comment (jsuarezgonz) [inline lib/src/text-input/TextInput.stories.tsx:17]: I think we can remove this 'console.log' and leave an empty function.

**Review by jsuarezgonz — COMMENTED** [review]
- It might be interesting to add an example to see how the disabled state affects the prefix and suffix.

- Add an example with a bigger icon in an action to check that it scales properly.

File: lib/src/text-input/TextInput.stories.tsx (hunk 1 above)
Comment (raquelarrojo) [reply lib/src/text-input/TextInput.stories.tsx]: Agree! Done!

**Comment by raquelarrojo** [comment]
> * It might be interesting to add an example to see how the disabled state affects the prefix and suffix.
> * Add an example with a bigger icon in an action to check that it scales properly.

Done!

**Review by jsuarezgonz — APPROVED** [review]

**Pull request merged**